src/hooks/pageFixture.ts

```diff
@@ -0,0 +1,6 @@
+import {Page} from '@playwright/test';
+
+export const pageFixture = {
```

Comment:
What is this object's responsibility?

Reply:
Instead of creating an object from Page every time, we are using this method.